pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -131,3 +131,109 @@ type AccountList struct {
 func init() {
 	SchemeBuilder.Register(&Account{}, &AccountList{})
 }
+
```

Comment:
Minor suggestion: Could put these into a separate file in the same package. But meh.